validator/fixtures/networks.json

```diff
@@ -880,7 +880,189 @@
             "country": "Côte d'Ivoire, Nigeria, Ghana, Uganda, Rwanda, Kenya",
             "number_of_stations": "171",
             "versions": [
-                30, 42
+                30, 42, 56
+            ]
+        }
+    },
+    {
+        "model": "validator.ismnnetworks",
```

Comment:
@sheenaze btw this is where the wrong network number comes from, I had to create 14 new entries here...
Also, the number of stations differs between versions, what is it actually used for?